stock_picking_whole_scrap/wizards/stock_picking_whole_scrap.py

```diff
@@ -0,0 +1,132 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models, _
+from odoo.addons import decimal_precision as dp
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_round, float_compare
+
+
+class StockPickingScrapLine(models.TransientModel):
+    _name = "wiz.stock.picking.scrap.line"
+    _description = "Wizard lines for picking whole scrap"
+    _rec_name = 'product_id'
+
+    product_id = fields.Many2one(
+        comodel_name='product.product',
+        string='Product',
+        readonly=True,
+    )
+    lot_id = fields.Many2one(
+        comodel_name='stock.production.lot',
+        string='Lot',
+        readonly=True,
+    )
+    package_id = fields.Many2one(
+        comodel_name='stock.quant.package',
+        string='Package',
+        readonly=True,
+    )
+    owner_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Owner',
+        readonly=True,
+    )
+    quantity = fields.Float(
+        string='Quantity',
+        digits=dp.get_precision('Product Unit of Measure'),
+        required=True,
+    )
+    uom_id = fields.Many2one(
+        comodel_name='uom.uom',
+        string='Unit of Measure',
+        readonly=True,
+    )
+    wizard_id = fields.Many2one(
+        comodel_name='wiz.stock.picking.scrap',
+        string='Wizard',
+    )
+    move_line_id = fields.Many2one(
+        comodel_name='stock.move.line',
+        string='Move Line',
+    )
+
+
+class StockPickingScrap(models.TransientModel):
+    _name = 'wiz.stock.picking.scrap'
+    _description = 'Picking Whole Scrap'
+
+    picking_id = fields.Many2one(comodel_name='stock.picking')
+    line_ids = fields.One2many(
+        comodel_name='wiz.stock.picking.scrap.line',
+        inverse_name='wizard_id',
+        string='Moves',
+    )
+    scrap_location_id = fields.Many2one(
+        comodel_name='stock.location',
+        string='Scrap Location',
+        domain=[('scrap_location', '=', True)],
+    )
+
+    @api.model
+    def default_get(self, fields):
+        if len(self.env.context.get('active_ids', list())) > 1:
+            raise UserError("You may only return one picking at a time!")
```

Comment:
```suggestion
            raise UserError("You may only scrap one picking at a time!")
```
and change translations accordingly